WordPress/Classes/Models/ReaderPost.h

```diff
@@ -50,6 +51,7 @@ extern NSString * const ReaderPostStoredCommentTextKey;
 @property (nonatomic, readonly, strong) NSURL *featuredImageURL;
 @property (nonatomic, strong) NSString *tags;
 @property (nonatomic, strong) ReaderAbstractTopic *topic;
+@property (nonatomic, strong) NSSet<ReaderCard *> *card;
```

Comment:
Are cards really one to many?  From my read of card related code and the changes here it seems like the answer is no and the one to many relationship is something we should sort out.  
To be clear, it would be a change out of scope for this PR.  I'm mostly just puzzled and wondering if I'm missing something that lets a post have multiple cards.

Reply:
Very good question indeed and one that I asked myself. They're not. I created this relationship and I think this was an oversight on my side. (SHAME!)

I'll address this in another PR. :)